5-appinfra/modules/env_baseline/outputs.tf

```diff
@@ -14,3 +14,27 @@
  * limitations under the License.
  */
 
+output "primary_instance" {
```

Comment:
What is the function of this output vs say `primary_instance_id` or `psc_dns_name`.  I recommend we minimize the number of unused outputs.

It's not obvious to me which of these is being passed to 6-appsource stage for configuration of the accounts and ledger services?

Reply:
removed them as they are not used in later stages.